### DISCUSSION TITLE
Red Eclipse 2 speedrunning.

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1277] [General]
I've been thinking about it, and figured that RE technically has 2 ways of speedrunning.
Normal race maps, as usual, try to beat your time (or others) as fast as possible.
And flag runs, from CTF, where you try to grab the enemy flag and figure out a way to score as fast as possible.
For doing flag runs, just start a Capture the Flag match as usual, remove all the bots, and keep going!
Save the demo of the match when it ends by typing /getdemo.
I'd like to see people's times by trying their best to set records of times on both types of runs, and sending their demos here, or videos, if possible.
Let's try to push the parkour to it's limtis?

I'll begin.
Flag run: Fortitude
Time:  0.07.952
Version: 2.0
Mutators? None.
Demo file: I don't remember if i have it or where it is.

https://user-images.githubusercontent.com/24359960/123345669-f6c03100-d52c-11eb-91c0-52e887086a1e.mp4

I'll begin.
Flag run: Octavus 
Time: 0.10.449
Version: 2.0
Mutators? None.
Demo file: I don't remember if i have it or where it is.

https://user-images.githubusercontent.com/24359960/123345907-7948f080-d52d-11eb-9736-578ecf4292d5.mp4

**Comment by robalni** [comment]
> Let's try to push the parkour to it's limtis?

Yes, I like pushing things to their limits! I didn't record any demos but I recorded videos.

**Fortitude**
Version: current master (0175d111)
Muts: no muts
Time: 7.543

[fortitude.webm](https://user-images.githubusercontent.com/1399719/179405888-327a6577-b414-4037-af68-699b7bd236f8.webm)

---

**Octavus**
Version: current master (0175d111)
Muts: no muts
Time: 9.450

[octavus.webm](https://user-images.githubusercontent.com/1399719/179406108-42fc7a8d-0037-4ba4-adf8-11a5ba1b1973.webm)

---

Your turn :)

**Reply by thevibingcat** [reply]
How do you get the old HUD and viewmodels?

**Reply by Jigoku** [reply]
It's recently been reintegrated in the development version. 
Make sure you're on the master/development branch by either:
* cloning the git repo
* switching to the beta branch on steam (via properties in your steam library)

**Comment by SniperGoth** [comment]
Ooooh, hell yeah, this is what i'm talking about!
I'll load up the dev version and play around later to see if i can get better times too.
Keep in mind that depending on the weight  you have (it is defined by the weapons you currently hold), you either get slower or faster, so take that in consideration.

**Comment by SniperGoth** [comment]
There it is, fortitude flag captured in 6.445 at current master, no mutators.
I'll do the other maps later.

https://user-images.githubusercontent.com/24359960/179421275-0862b7c0-30bd-43b1-84b7-21c90e766b32.mp4

**Comment by robalni** [comment]
With lighter weapons this time.

**Fortitude**
Version: master 0175d111
Muts: none
Time: 5.941

[fortitude-5-941.webm](https://user-images.githubusercontent.com/1399719/179474764-ccae128b-aeb1-4650-85b4-1fe2f048f164.webm)

---

Edit: I just discovered that you get a pretty big speed boost from sword2 in this game so now it's possible to cut off another second.

**Fortitude**
Version: master 0175d111
Muts: none
Time: 4.875

[fortitude-4-875.webm](https://user-images.githubusercontent.com/1399719/179497113-d1f75f6d-71cd-4bd6-986b-ec9038b7c100.webm)

**Reply by SniperGoth** [reply]
Wow. You literally flew around the map, hahah.
This is where the fun begins.
